frontend/packages/dev-console/src/components/pipelineruns/PipelineRunResourceList.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { ListPage } from '@console/internal/components/factory';
+import { DevPreviewBadge } from '@console/shared';
+import {
+  pipelineRunFilterReducer,
+  pipelineRunStatusFilter,
+} from '../../utils/pipeline-filter-reducer';
+import { PipelineRunModel } from '../../models';
+import PipelineRunsList from './PipelineRunList';
+
+const filters = [
```

Comment:
We are duplicating filters. `frontend/packages/dev-console/src/components/pipelines/PipelineRuns.tsx` has the same filters.

Reply:
Making sure, the same filter is used in both places